open_instruct/grpo_fast.py

```diff
@@ -1391,6 +1396,42 @@ def data_preparation_thread(
                 datasets = [datasets[i] for i in stop_idxes]
                 finish_reasons = [finish_reasons[i] for i in stop_idxes]
 
+            if args.fill_completions:
```

Comment:
Could we add a timer on this block of code to see how much time it takes? Should be easy with the decorator.